### PR TITLE
fix meeting time and URL

### PR DESCRIPTION
**Pull request by JimBugwadia**

### CODE REVIEW

File: wg-policy/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Provide an overall architecture that describes both the current policy related i
 * SIG Storage
 
 ## Meetings
-* Regular WG Meeting: [Wednesdays at 16:00 PT (Pacific Time)](https://zoom.us/j/7375677271) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=16:00&tz=PT%20%28Pacific%20Time%29).
+* Regular WG Meeting: bi-weekly [Wednesdays at 08:00 AM PT (Pacific Time)](https://zoom.us/my/cncfsigsecurity) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=16:00&tz=PT%20%28Pacific%20Time%29).
```
Comment (mrbobbytables) [inline wg-policy/README.md:23]: This should be separate from the CNCF group and managed under a kubernetes owned zoom account.

Comment (JimBugwadia) [reply wg-policy/README.md:23]: Here's where I got that from: https://docs.google.com/document/d/1ihFfEfgViKlUMbY2NKxaJzBkgHh-Phk5hqKTzK-NEEs/edit. What zoom should be used instead?

**Comment by JimBugwadia** [comment]
@cblecker - how can we correct the meeting information on the community web page?

**Pull request closed without merging**